packages/bpk-component-drawer/readme.md

```diff
@@ -46,13 +46,15 @@ class App extends Component {
           <BpkButton onClick={this.onOpen}>Open drawer</BpkButton>
         </div>
         <BpkDrawer
+          id="my-drawer"
           isOpen={this.state.isOpen}
           onClose={this.onClose}
           title="Drawer title"
           closeLabel="Close drawer"
           getApplicationElement={() =>
             document.getElementById('application-container')
           }
+          hideTitle
```

Comment:
Why showcase the version with hidden title in the docs?

Reply:
To show it's there? No better reason than that, I can remove it if you want.

Reply:
I think we should showcase the version with non-hidden title here

Reply:
Alright, I'll remove the line.